forge-gui/res/cardsfolder/upcoming/scholar_of_new_horizons.txt

```diff
@@ -0,0 +1,13 @@
+Name:Scholar of New Horizons
+ManaCost:1 W
+Types:Creature Human Scout
+PT:1/1
+K:etbCounter:P1P1:1
+A:AB$ ChangeZone | Cost$ T RemoveAnyCounter<1/Any/Permanent.YouCtrl/a permanent you control> | Origin$ Library | Destination$ Library | ChangeType$ Land.Plains | ChangeNum$ 1 | RememberChanged$ True | Reveal$ True | Shuffle$ False | SubAbility$ ToBattlefieldTapped | SpellDescription$ Search your library for a Plains card and reveal it. If an opponent controls more lands than you, you may put that card onto the battlefield tapped. If you don't put the card onto the battlefield, put it into your hand. Then shuffle.
```

Comment:
probably `StackDescription$ SpellDescription` and be done with it... some of the other lines might need `StackDescription$ None`

Reply:
Probably `ChangeType$ Plains` – you know WotC, they'll be making Plains cards that aren't lands at some point